public/theme/skeleton/partials/_master.twig

```diff
@@ -7,9 +7,9 @@
         {# make sure we always display a proper title: The record's title if there is one, appended with the
            sitename. If there is no title, we append the sitename with the payoff, if there is one. #}
         <title>
-            {%- if record.hasField('title') %}{{ record.title|striptags }} | {% endif -%}
+            {%- if record is defined and record.title %}{{ record.title|striptags }} | {% endif -%}
```

Comment:
no need to strip tags :) or this one has html allowed?

Reply:
> this one has html allowed

It's a `type: text`, so it doesn't by default, but you don't know how somebody else changes it. :-)

Reply:
then somebody else has to update template too :)